### PR TITLE
[Localization][DE] Translation First  Test of Pontoon PR

### PR DESCRIPTION
**Pull request by CodeTappert**
## What are the changes the user will see?

More localized strings

## Why am I making these changes?

To test pontoon and to localize missing stuff

## What are the changes from a developer perspective?

See Why am I making these changes?

### Screenshots/Videos

Maybe

## How to test the changes?

Play in the languages were it was changed

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [ ] If I have text, did I make it translatable and add a key in the English locale file(s)?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/de/modifier-type.json [hunk 1]
```diff
@@ -416,12 +420,25 @@
...
+    "MYSTERY_ENCOUNTER_MACHO_BRACE": {
+      "name": "Machoschiene",
+      "description": "Das Besiegen eines Pokémon gewährt dem Besitzer einen Machoschiene-Stapel. Jeder Stapel steigert die Werte leicht, mit einem zusätzlichen Bonus bei maximalen Stapeln."
+    },
+    "MYSTERY_ENCOUNTER_OLD_GATEAU": {
+      "name": "Spezialität",
+      "description": "Erhöht den {{stats}}-Wert des Trägers um {{statValue}}."
+    },
+    "MYSTERY_ENCOUNTER_GOLDEN_BUG_NET": {
+      "name": "Golden Bug Net",
```
Comment (CodeTappert) [inline src/locales/de/modifier-type.json:439]:
```suggestion
      "name": "Goldenes Käfersammlernetz",
```

**Comment by CodeTappert** [comment]
image

No Screenshots i guess :D

**Review by MokaStitcher — APPROVED** [review]

**Review by Adri1 — APPROVED** [review]

**Review by SangaraSorama — APPROVED** [review]
I do **not** count as a dev, in case it matters

**Comment by flx-sta** [comment]
> I do **not** count as a dev, in case it matters

No DEV changes thus it's fine

**Pull request merged**